Altis_Life.Altis/core/fn_setupStationService.sqf

```diff
@@ -1,6 +1,8 @@
+#define ALTIS_TANOA(var1,var2) if (worldName isEqualTo "Altis") then {var1} else {var2}
```

Comment:
Aren't we using `==` instead of `isEqualTo` because it's **not** case sensitive?